scripts/jobExecCondor.sh

```diff
@@ -27,27 +33,33 @@ stageOut() {
 		$ECHO "-x args       \tany arguments to pass to xrdcp/gfal-copy (should be quoted)"
 		$ECHO "-g            \tUse gfal-copy rather than xrdcp"
 		$ECHO "-q            \tquiet (don't print any messages)"
+		$ECHO "-r            \tremove local file if successfully copied"
```

Comment:
I don't think this is a bad thing, but what was your reasoning for putting it in? Just to help keep the wn in good shape?

Reply:
Previously, I had been doing the "rm file after successfully copied" part in each `step2.sh` script. This centralizes that behavior, which makes it easier to control: for intermediate jobs, we don't want to remove the file, but rather we want to keep it local for the next job to consume. `stageOut` now handles that.

The deeper reason the behavior is needed at all is because `Should_Transfer_Files = YES` in the JDL will bring back any new files created in the top-level directory on the worker node.